changelogs/unreleased/gh-7362-net-box-async-ping.md

```diff
@@ -0,0 +1,3 @@
+## feature/core
+
+* Introduced `is_async` option for `conn:ping()` (gh-7362).
```

Comment:
What is `conn` here? Lets be more specific and mention that it is a netbox connection.